qe_metrics/__init__.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Module building qe_metrics cli"""
+
+import click
+
+from qe_metrics.commands.init_db import init_db
+
+
+@click.group()
```

Comment:
This was modeled after firewatch which was modeled after MPS Utils. I can change it, I have only done it this way because this is what was used as an example: https://gitlab.cee.redhat.com/mpqe/mps/mps-utils/-/tree/develop/mps_utils/cli?ref_type=heads